### PR TITLE
DOCS-3874 Browser Test, API Tests, and Browser Test Steps Edits 

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Updates Browser Test doc pages with descriptions for local and global variables, addresses consistency nits across API and multistep API test pages, and documents the **Hide and obfuscate variable values** option.

### Motivation

DOCS-3874

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/multistep.md [hunk 1]
```diff
@@ -91,18 +93,11 @@ By default, a maximum of 10 steps can be created. To increase this limit, reach
 
    {{% tab "Request Body" %}}
 
-   * **Body type**: Select the type of the request body (`text/plain`, `application/json`, `text/xml`, `text/html`, `application/x-www-form-urlencoded`, or `None`) you want to add to your HTTP request.
+   * **Body type**: Select the type of the request body (`text/plain`, `application/json`, `text/xml`, `text/html`, `application/x-www-form-urlencoded`, `GraphQL`, or `None`) you want to add to your HTTP request.
```
Comment (HugoPct) [inline content/en/synthetics/multistep.md:96]: nice catch !

Comment (HugoPct) [reply content/en/synthetics/multistep.md:96]: can you add it the the 
- HTTP tests page
- HTTP requests for browser tests in /browser_tests/actions/ page
too please

File: content/en/synthetics/multistep.md [hunk 2]
```diff
@@ -244,10 +228,16 @@ You can create local variables by clicking **Create Local Variable** at the top
...
-: Generates a date in one of our accepted formats with a value of the date the test is initiated + `n` days.
+: Generates a date in one of Datadog's accepted formats with a value corresponding to the UTC date the test is initiated + or - `n` days.
 
 `{{ timestamp(n, unit) }}` 
-: Generates a timestamp in one of our accepted units with a value of the timestamp the test is initiated at +/- `n` chosen unit.
+: Generates a timestamp in one of Datadog's accepted units with a value corresponding to the UTC timestamp the test is initiated at + or - `n` units.
+
+To obfuscate local variable values in test results, select **Hide and obfuscate variable value**. Once you have defined the variable string, click **Add Variable**.
+
+### Extract variables
```
Comment (HugoPct) [inline content/en/synthetics/multistep.md:238]: can we keep this section as first section below ### Create local variables
Extracting variables from a step to reuse it in following steps is a key strength of multistep tests

**Review by HugoPct — CHANGES_REQUESTED** [review]

**Comment by HugoPct** [comment]
For API tests pages, I've commented the [dns_tests.md] only but the comments apply for all the API test pages

File: content/en/synthetics/api_tests/dns_tests.md [hunk 3]
```diff
@@ -124,7 +124,7 @@ For more information, see [Using Synthetic Test Monitors][9].
 
 ### Create local variables
 
-You can create local variables by clicking **Create Local Variable** at the top right hand corner of your test configuration form. You can define their values from one of the below available builtins:
+To create a local variable, click **Create Local Variable** at the top right hand corner. You can select one of the following available builtins:
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/synthetics/api_tests/dns_tests.md:127]: I was thinking about removing the repetition of "You can..." from the first and second sentence mostly, but also looking in-app, the section for variables is pretty front and center as an option for users.

**Comment by kayayarai** [comment]
Opened https://datadoghq.atlassian.net/browse/DOCS-3882 to track editorial review

File: content/en/synthetics/browser_tests/actions.md [hunk 4]
```diff
@@ -342,37 +342,30 @@ You can also optionally extract a variable from the response of your HTTP reques
...
+Once created, this variable can be used in the [following steps](#use-variables) of your browser test.
 
-To see all available variables, type `{{` in the input field and choose your variable of interest.
+## Use variables
 
-### On automatically recorded steps
+To see all available variables on manually added steps, type `{{` in the input field.
 
-If you want to record a step leveraging a variable, you can use the little hand on your variable box:
+If you want to use a variable on automatically recorded steps, click **Inject this variable** icon:
```
Comment (alai97) [inline content/en/synthetics/browser_tests/actions.md:359]:
```suggestion
If you want to use a variable on automatically recorded steps, click the **Inject this variable** icon:
```

File: content/en/synthetics/browser_tests/_index.md [hunk 5]
```diff
@@ -80,66 +79,95 @@ Define the configuration of your browser test.
...
+You can use the [global variables defined in **Settings**][5] in the **Starting URL** and **Advanced Options** of your browser test details, as well as in your test recording to define local variables. To display a list of available variables, type `{{` in the desired field.
 
-{{< img src="synthetics/browser_tests/using_variables_browser.mp4" alt="Using Variables in Browser Tests" video="true" width="80%" >}}
+{{< img src="synthetics/browser_tests/recording_global_variable.mp4" alt="Defining a local variable from global variables" video="true" width="90%" >}}
 
-To use global variables in your browser test's recording, navigate to your recorded steps and click **+ Variables** below the **Start Recording** button. Select **Global Variable** from the dropdown menu. 
+Define the variables you want to incorporate into the user journey before you start recording.
 
-{{< img src="synthetics/browser_tests/available-global-variables.png" alt="Available Global Variables" style="width:50%;" >}}
+{{< img src="synthetics/browser_tests/recording_inject_variable.mp4" alt="Injecting a local variable into a field during a browser recording" video="true" width="90%" >}}
```
Comment (HugoPct) [inline content/en/synthetics/browser_tests/_index.md:138]: this new vid is very cool!

File: content/en/synthetics/browser_tests/_index.md [hunk 6]
```diff
@@ -80,66 +79,95 @@ Define the configuration of your browser test.
...
-    | `{{^is_alert}}`            | Show unless monitor alerts                                          |
-    | `{{#is_recovery}}`         | Show when monitor recovers from either ALERT   |
-    | `{{^is_recovery}}`         | Show unless monitor recovers from either ALERT |
-    | `{{#is_renotify}}`         | Show when the monitor renotifies.   |
-    | `{{^is_renotify}}`         | Show unless the monitor renotifies. |
-    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).   |
-    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).  |
+    | `{{#is_alert}}`            | Show when the monitor alerts.                                       |
+    | `{{^is_alert}}`            | Show unless the monitor alerts.                                     |
+    | `{{#is_recovery}}`         | Show when the monitor recovers from alert.                          |
```
Comment (HugoPct) [inline content/en/synthetics/browser_tests/_index.md:161]:
```suggestion
    | `{{#is_recovery}}`         | Show when the monitor recovers from `alert`.                          |
```

File: content/en/synthetics/browser_tests/_index.md [hunk 7]
```diff
@@ -80,66 +79,95 @@ Define the configuration of your browser test.
...
+    | `{{^is_recovery}}`         | Show unless the monitor recovers from alert.                        |
```
Comment (HugoPct) [inline content/en/synthetics/browser_tests/_index.md:162]:
```suggestion
    | `{{^is_recovery}}`         | Show unless the monitor recovers from `alert`.                        |
```

**Review by HugoPct — APPROVED** [review]
added 2 suggestions otherwise good to go for me

File: content/en/agent/troubleshooting/agent_check_status.md [hunk 8]
```diff
@@ -147,4 +147,3 @@ sudo journalctl -u dd-agent.service
 [1]: /integrations/
 [2]: /agent/troubleshooting/send_a_flare/
 [3]: https://github.com/DataDog/datadog-agent/blob/master/docs/agent/changes.md#service-lifecycle-commands
-[4]: /integrations/
```
Comment (urseberry) [inline content/en/agent/troubleshooting/agent_check_status.md:150]: Please revert this change. Removing the link breaks a link above, in the Linux section, on the Agent v5 tab.

**Review by urseberry — COMMENTED** [review]
I need to sign off for now, but I'm submitting these comments so they don't get lost. Will finish the review later.

**Pull request closed without merging**